src/metrics/family.rs

```diff
@@ -192,16 +202,16 @@ impl<S: Clone + std::hash::Hash + Eq, M> Family<S, M> {
     }
 }
 
-impl<S, M> Clone for Family<S, M> {
+impl<S, M, C: MetricConstructor<M> + Clone> Clone for Family<S, M, C> {
```

Comment:
```suggestion
impl<S, M, C: Clone> Clone for Family<S, M, C> {
```

Is this bound needed?